analyses/cell-type-wilms-tumor-06/notebook-template/01-clustering.Rmd

```diff
@@ -0,0 +1,768 @@
+---
+title: "Evaluation of clustering parameters"
+author: maud-p
+date: '`r format(Sys.time(), "%B %d, %Y %H:%M:%S %Z")`'
+output:
+  html_document: 
+    toc: yes
+    toc_float: yes
+    code_folding: hide
+    highlight: pygments
+    df_print: kable
+params:
+  sample_name: ~
+  sample_path: ~
+  out_rds_path: ~
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+	echo = TRUE,
+	message = FALSE,
+	warnings = FALSE,
+	error = TRUE
+)
+```
+
+
+## Introduction
+
+This a template for an analysis notebook using RMarkdown.
+
+In this notebook, we will set up parameters in the seurat workflow [normalization --> reduction --> clustering] for one Wilms tumor sample (SCPCS000169) of the Wilms Tumor dataset (SCPCP000006). 
+
+This correspond to the step 2 of the proposed analysis: clustering of cells across a set of parameters for few samples
+
+- https://github.com/AlexsLemonade/OpenScPCA-analysis/discussions/635
+- https://github.com/AlexsLemonade/OpenScPCA-analysis/issues/671
+
+In order to assess the clustering quality, we look into some marker genes, pathways enrichment and label transfer.
```

Comment:
Again, I would argue that you're not really assessing quality (that would involve an in-depth investigation of clustering parameter values and assessing robustness of clustering to different parameters), but more exploring results from clustering, so I might rephrase this a smidge.